flutter-studio/src/io/flutter/project/FlutterProjectSystem.java

```diff
@@ -65,7 +65,7 @@ public AndroidModuleSystem getModuleSystem(@NotNull Module module) {
     return gradleProjectSystem.getModuleSystem(module);
   }
 
-  @Override
+  @SuppressWarnings("override")
```

Comment:
I'm guessing this is an override only on one platform, right?

Reply:
It is on several, but yeah, not all. This method got removed in the latest AS sources but is still required to compile older versions.